core/players-handbook-2024/classes/class-barbarian.xml

```diff
@@ -0,0 +1,459 @@
+		</description>
+		<sheet alt="Persistent Rage" usage="1/Long Rest">
+			<description>When you roll Initiative, you can regain all expended uses of Rage. 
+			In addition, your Rage is so fierce that it now lasts for 10 minutes without you needing to do anything to extend it from round to round. Your Rage ends early if you have the Unconscious condition or don Heavy armor.</description>
+		</sheet>
+	</element>
+
+	<element name="Level 17: Improved Brutal Strike" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_BARBARIAN_IMPROVED_BRUTAL_STRIKE_COUNT">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_BARBARIAN_24_IMPROVED_BRUTAL_STRIKE_COUNT</requirements>
+		<description>
+			<p>The extra damage of your Brutal Strike increases to 2d10. In addition, you can use two different Brutal Strike effects whenever you use your Brutal Strike feature.</p>
+		</description>
+		<sheet display="false" />
+	</element>
+
+	<element name="Level 18: Indomitable Might" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_BARBARIAN_INDOMITABLE_MIGHT">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_BARBARIAN_24_INDOMITABLE_MIGHT</requirements>
+		<description>
+			<p>If your total for a Strength check or Strength saving throw is less than your Strength score, you can use that score in place of the total.</p>
+		</description>
+		<sheet alt="Indomitable Might">
+			<description>If your total for a Strength check or Strength saving throw is less than {{strength:score}}, you can use {{strength:score}} place of the total.</description>
+		</sheet>
+	</element>
+
+	<element name="Level 19: Epic Boon" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_BARBARIAN_EPIC_BOON">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_BARBARIAN_24_EPIC_BOON</requirements>
+		<description>
+			<p>You gain an Epic Boon feat or another feat of your choice for which you qualify. Boon of Irresistible Offense is recommended.</p>
+		</description>
+		<sheet display="false">
+		</sheet>
+		<rules>
+			<select type="Feat" name="Epic Boon (Barbarian 19)" level="19" />
+		</rules>
+	</element>
+
+	<element name="Level 20: Primal Champion" type="Class Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_CLASS_FEATURE_BARBARIAN_PRIMAL_CHAMPION">
+		<requirements>!ID_INTERNAL_FEATURE_REPLACEMENT_BARBARIAN_24_PRIMAL_CHAMPION</requirements>
+		<description>
+			<p>You embody primal power. Your Strength and Constitution scores increase by 4, to a maximum of 25.</p>
+		</description>
+		<sheet display="false">
+			<description>You embody primal power. Your Strength and Constitution scores increase by 4, to a maximum of 25.</description>
+		</sheet>
+		<rules>
+			<stat name="strength" value="4" />
+			<stat name="constitution" value="4" />
+			<stat name="strength:max" value="4" bonus="ability" />
+			<stat name="constitution:max" value="4" bonus="ability" />
```

Comment:
since feature adds it to a maximum of 25, stats that increase ability's max should give 5. 
Changing bonus is not a fix or anything, i just think it's better to give these a unique names.
```suggestion
			<stat name="strength:max" value="5" bonus="primal champion" />
			<stat name="constitution:max" value="5" bonus="primal champion" />
```